### DISCUSSION TITLE
Change InputLeftElement and InputRightElement color when input has be focus

### DISCUSSION THREAD

**Discussion by jhosepharaujo** [chakra-ui/chakra-ui#2046] [General] [answered]
How to change the color of InputLeftElement and InputRightElement when the input is in focus, similar to the image?

![input focus](https://user-images.githubusercontent.com/2037683/93260063-34f71b80-f777-11ea-9fc8-af9d272daffb.PNG)

**Comment by ljosberinn** [comment, marked as answer]
We recently had a similar question [on discord](https://discordapp.com/channels/660863154703695893/660865137137156146/757990659411017870). You may leverage the `sx` prop in combination with the underlying static classNames chakra comes with on every element:

```js
<InputGroup
  sx={{
    '.chakra-input:focus + .chakra-input__right-element .chakra-icon': {
      color: 'red.500'
  }}}
```

This works fine for the right element, haven't tried for the left but I'm sure a similar solution works.
